versioned_docs/version-8.3/guides/setup-client-connection-credentials.md

```diff
@@ -14,7 +14,6 @@ Here, we'll set up client connection credentials to create, name, and connect yo
 Currently, Camunda 8 SaaS supports the following scopes:
 
 - Zeebe - Access to the [Zeebe gRPC](/apis-tools/grpc.md) API.
-- Tasklist - Access to the [Tasklist GraphQL](/apis-tools/tasklist-api/tasklist-api-overview.md) API.
```

Comment:
As we still support Tasklist V1, we should add here the reference to Tasklist API (https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/tasklist-api-rest-overview/)